Content.Tests/DMProject/Tests/Statements/Control Flow/blocks_1.dm

```diff
@@ -0,0 +1,11 @@
+
+//# issue 466
+
+/proc/RunTest()
+	var/i = 0
+	label_name:
+		if(i < 1)
+			i += 1
+			goto label_name
+
+	ASSERT(i == 1)
```

Comment:
This would pass even if `goto label_name` never did anything. It would be better to check if the code runs at least twice.
```suggestion
		if(i < 2)
			i += 1
			goto label_name

	ASSERT(i == 2)
```